README.md

```diff
@@ -248,6 +248,8 @@ features to your store that are not (or not yet) part of the core project.
 
 A list can be found at [extensions.solidus.io](http://extensions.solidus.io/).
 
+If you want to write an extension for Solidus, you can use the [solidus\_cmd](https://www.github.com/solidusio/solidus_cmd.git) gem.
```

Comment:
There is a typo in the link text. Should be `solidus_cmd`, without the backslash.

Reply:
The character is escaped: https://daringfireball.net/projects/markdown/syntax#backslash